asset/boltdb_manager.go

```diff
@@ -241,6 +243,14 @@ func (b *boltDBAssetManager) expandWithDuration(tmpFile *os.File, asset *corev2.
 	}))
 	defer timer.ObserveDuration()
 
+	assetSHA := asset.Sha512
+	CacheDir := viper.GetString(FlagCacheDir)
+	fullPath := filepath.Join(CacheDir, assetSHA)
+
+	if err := CleanUp(fullPath); err != nil { //fix for git issue 5009
+		logger.Printf("error cleaning up the SHA dir: %s", err)
```

Comment:
It would be nice if more context information such as the SHA  and full path could be provided with the log message. You can use `WithFields` and `WithError` .

Additionally please use an explicit log level such as `Errorf` or `Warnf` instead of using the generic `Printf`. In this case I believe warning would be an appropriate log level.

You can see an example at https://github.com/sensu/sensu-go/blob/76c7a167e572bb370438e38b678fa19c3d973de1/backend/pipeline/filter/legacy.go#L126